src/rest-errors/index.js

```diff
@@ -20,6 +22,8 @@ module.exports = {
    * @return {RestError}
    */
   toRestError(err) {
+    debug(err);
```

Comment:
Hmm, not the right place for this. Better to do it here : https://github.com/SylvainEstevez/Restypie/blob/master/src/resources/lib/abstract-resource.js#L1246